Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -6,34 +6,39 @@ public enum ActorStates
 {
 	Grounded = 0,
 	Jumping,
+	Falling,
 	Rolling,
 	Climbing
 }
 
 public class ActorPhysics : ActorComponent
 {
+	private new Transform transform;
+	private new Rigidbody rigidbody;
+
 	// States
+	#region States
 	public delegate void ActorStateMethod();
 
+	[ReadOnlyAttribute, SerializeField]
```

Comment:
Can we name attributes without the word Attribute in them? It would be preferable not to since it can be assumed, but I might recall that being needed...